bentoml/_internal/bento/build_dev_bentoml_whl.py

```diff
@@ -1,9 +1,14 @@
 import os
 import uuid
-import shutil
 import logging
 import importlib
 import importlib.util
+from sys import version_info
+
+if version_info >= (3, 8):
+    from shutil import copytree
+else:
+    from backports.shutil_copytree import copytree
```

Comment:
why do we need this?

Reply:
for 3.7, `dir_exists_ok` does not exist.